lib/WebService/MinFraud/Record/Warning.pm

```diff
@@ -25,7 +25,7 @@ has input => (
 
 1;
 
-# ABSTRACT: Contains data for the warning record returned from a web service query
+# ABSTRACT: Contains data for a warning record returned from a web service query
```

Comment:
Does this contain data for the warning record or is it the actual warning record?